.github/workflows/scripts_tests.yml

```diff
@@ -34,6 +34,18 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
           fetch-depth: 0
 
+      - name: Rebase
+        continue-on-error: true
+        env:
+          BASE_REF: ${{ github.base_ref }}
+          PR_HEAD: ${{ github.event.pull_request.head.sha }}
+        run: |
+          export ZEPHYR_BASE=$PWD
```

Comment:
is this line needed for the rebase?

Reply:
It's not, so used to see it everywhere I just blindly copy pasted it :-) dropped